### ISSUE TITLE
new method for excavation unit: "three-tier"

### ISSUE THREAD

**Issue by urapadmin** [urapadmin/kiosk#663]
And that method opens an excavation unit that does not offer loci but features. We might need a better name for it. It will be basically the same kind of unit like the recording unit except that instead of loci it allows to create or connect feature units. It will have dayplans, narratives and the usual unit description. 

It will NOT have an invisible locus because it cannot have collected material assigned directly. The collected material is assigned to the feature units.

As other excavation units, too, it cannot be tagged.

**Comment by urapadmin** [comment]
see also #664

**Comment by urapadmin** [comment]
and should we call the excavation unit method we are currently using "locus recording"?

**Comment by lbestock** [comment]
Hm. I don't really think the new type of unit could ever be used for excavation currently. One could not excavate without locus recording now, so distinguishing between "locus recording" and "excavation" makes no sense. In a way this type of unit might always move opposite of how our other units do. Other units are defined and then things are found within them. These units are defined after things are found, as groups of found things. (That is true for Wheeler's blocks even, not just because we're putting in legacy data. They were used to describe what he had already found.) I can think of other uses for that. So if we survey the Western desert and we have our survey units and in them we find and describe 100 features, and then on having the whole picture in front of us it becomes clear that some of those features cluster and are a cemetery, and others cluster and are a settlement, I would want to apply this type of unit to them. That's what Wheeler did - he didn't know he was digging barracks until he had dug them. He couldn't have defined Block III until he had dug it. But hm again, the reason we thought we needed this was as a way to organize narratives and dayplans in particular. And with NEW material, if the unit isn't defined in the first place then narratives are created where?

Basically I think (and this is where the tag confusion comes in) we're talking about a unit of analysis not a unit of observation/recording. And they are confused in the Wheeler case because our own process is backwards: the recording in our system is coming after the analysis.

Uh. In fact, where would I keep narrative notes about my pedestrian survey and the features I assign? Those types of units have descriptions but not narratives. And what we're talking about here are narratives that bridge multiple such units. So site notes? Again we're back to needing site notes as narratives, which we abandoned and built new structure precisely to avoid? With survey units the description field was sometimes but not always used in this way at Uronarti.

**Comment by urapadmin** [comment]
hm that was longish. And does not give me a clue what to name them, which is my real concern here. But let me try to reply:
- If it is a problem that there are no narratives in survey units (or feature units for that matter) we can add them at any time. But due to the nature of the survey we were pretty confident that a single text field (the description) sufices.
- that is why I come back to looking at this as archaeological data coming out of excavation and that was the reason why I wanted to make this an excavation unit (it has been dug, not it is going to be dug). I would definitely use the current excavation form except for the difference between loci and features)
- I was not suggesting to put "locus recording" and "excavation" as mutually exclusive options, my suggestion is that this is an excavation unit (so type is excavation)  but the method would be "feature recording" instead of what we currently call - redundantly - "excavation"
- and because that redundancy does not make much sense we could call the current method "locus recording". So in type we choose "excavation" and in method we choose "locus recording" for our usual way or the new "feature recording" for the new way.

**Comment by lbestock** [comment]
Ah I understand now. And yes, now I think that your solution sounds the right one. 

When description was used for survey (they got better about it over the course of the season) it tends to be long but also very much a mixture of what description and narrative would be for an excavation unit. But yes I agree it still suffices.

**Comment by urapadmin** [comment]
### v9.2.2
The unit type excavation can now be combined with method feature recording. The recording form is exactly the same (in fact it is the same) as the simple excavation form but in this case it will show "features" instead of loci.
- [ ] Make sure, that it shows the term for features as configured in labels/term_for_features
- [ ] You should be able to add new features
- [ ] the rest of the recording is the same as in simple excavation, so that does not really need to be tested. Just poke around a bit.
